server_src/utils/git.utils.js

```diff
@@ -0,0 +1,39 @@
+const simpleGit = require("simple-git");
+const git = simpleGit();
+
+function returnCurrentGitStatus() {
+  return git.status();
+}
+
+function isBranchUpToDate(status) {
+  return status.behind === 0;
+}
+
+function isBranchInfront(status) {
+  return status.ahead !== 0;
+}
+
+function doesBranchContainModifiedFiles(status) {
+  if (status.modified.length !== 0) {
+    return status.modified;
```

Comment:
Return type inconsistent. Should be boolean, so convert with !!

Reply:
No because I want the modified file list